test/apps/iodemo/io_demo.cc

```diff
@@ -948,6 +948,20 @@ class P2pDemoCommon : public UcxContext {
         validate(conn, msg, iomsg_size);
     }
 
+    static inline void handle_io_msg(const iomsg_t *msg, UcxConnection *conn) {
```

Comment:
maybe better call `check_conn_id`, which is the only thing this func does

Reply:
fixed